pkg/fanal/analyzer/language/nodejs/yarn/testdata/latest-version/yarn.lock

```diff
@@ -0,0 +1,20 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+debug@latest:
+  version "4.3.5"
+  resolved "https://registry.npmjs.org/debug/-/debug-4.3.5.tgz"
+  integrity sha512-pt0bNEmneDIvdL1Xsd9oDQ/wrQRkXDT4AUWlNZNPKvW5x/jyO9VFXkJUP07vQ2upmw5PlaITaPKc31jK13V+jg==
+  dependencies:
+    ms "2.1.2"
+
+js-tokens@npm:latest:
```

Comment:
Why is it `latest`? package.json specifies `"js-tokens": "^9.0.0"`. I thought this line would be `js-tokens@^9.0.0`.

Reply:
I updated this manually to test that new logic worked correctly.

Reply:
I think it's better to keep using an original file as much as possible so we can understand how it actually works.

Also, instead of using `latest` in the package ID, what if we return package patterns somehow and do an exact match? Or, we may want to pass package.json to the yarn lock parser.

Reply:
> I think it's better to keep using an original file as much as possible so we can understand how it actually works. 

okay, i will remove this.

> Also, instead of using latest in the package ID, what if we return package patterns somehow and do an exact match? Or, we may want to pass package.json to the yarn lock parser.

Hm... i will think about this.

Reply:
> what if we return package patterns somehow and do an exact match

@knqyf263 I implemented your idea. Take a look, when you have time.